scripts/install_miniconda.sh

```diff
@@ -46,3 +46,4 @@ fi
 export PATH="$INSTALL_FOLDER/bin:$PATH"
 echo "Adding $INSTALL_FOLDER to PATH.  Consider adding it in your .rc file as well."
 conda update -q -y conda
+pip uninstall -y numpy
```

Comment:
I would add a comment that this is installing TravisCI's version of numpy. At a glance its confusing why numpy would be uninstalled

Reply:
Thanks Ravin! I'll add it once I get back to my computer in a few hours